scripts/lib/api/processHtml.ts

```diff
@@ -122,6 +122,13 @@ export function removePermalinks($main: Cheerio<any>): void {
   }
 }
 
+export function removeMatplotlibFigCaptions($main: Cheerio<any>): void {
```

Comment:
This function needs to be called inside `processHtml`. I recommend putting it after `removeDownloadSourceCode($main);`

Reply:
Done.